app/screens/home/DetailsScreen.tsx

```diff
@@ -430,9 +430,9 @@ const DetailsScreen = ({ route, navigation }: any): any => {
     let highlightedContent = content; // Initialize with original content
    
     query.forEach((item) => {
-        const regex = new RegExp(`${item}`, 'gi');
+        const regex = new RegExp(`\\b(${item.trim()}\\s*)`, 'gi');

```

Comment:
Can you try using this code to remove extra space?

const regex = new RegExp(`\\b(${item.trim()})\\b`, 'gi');

Reply:
yes @Akhror , I already tried this as well it's also not working.

Reply:
Does this help then?

const regex = new RegExp(`\\b${item.trim()}\\b`, 'gi');

Reply:
let me try with this

Reply:
@Akhror It is still not working.

Reply:
Ok last try:
const regex = new RegExp(`\\s*${item.trim()}\\s*`, 'gi');

Reply:
@Akhror I checked and it creates issue, it merge searched keyword with next text.

Reply:
Ok, let's keep as is then.